en/battle.json

```
@@ -115,5 +115,6 @@
   "ppReduced": "It reduced the PP of {{targetName}}’s\n{{moveName}} by {{reduction}}!",
   "mysteryEncounterAppeared": "What’s this?",
   "battlerTagsHealBlock": "{{pokemonNameWithAffix}} can’t restore its HP!",
-  "battlerTagsHealBlockOnRemove": "{{pokemonNameWithAffix}} can restore its HP again!"
+  "battlerTagsHealBlockOnRemove": "{{pokemonNameWithAffix}} can restore its HP again!",
+  "pokemonTerastallized": "{{pokemonNameWithAffix}} terastallized into a {{type}} type!"
```

```suggestion
  "pokemonTerastallized": "{{pokemonNameWithAffix}} transformed into the {{type}} type!"
```

[Official game text](https://abcboy101.github.io/poke-corpus/#id=sv.btl_set.BTL_STRID_SET_ChangePokeType)

---

This is more for stuff like soak right? Could still work though, not fussed on the exact string. Maybe `terastallized into the`?

---

We already have three versions of this sentence in the locales, would be nice for is on to be more original
So I would advocate for the original one wrote by Xavion

---

Ah, you're right.
In that case, "Terastallized into the" would be it.